MobileCenter/MobileCenterTests/MSCustomPropertiesTests.m

```diff
@@ -47,17 +47,31 @@ - (void)testKeyValidate {
   
   // Then
   assertThat([customProperties properties], hasCountOf(0));
-  
+
+  // Too long key.
+  // When
+  NSString *tooLongKey = [@"" stringByPaddingToLength:129 withString: @"a" startingAtIndex:0];
```

Comment:
That's also here. 😃 
https://github.com/Microsoft/mobile-center-sdk-ios/pull/546/files#diff-ecf1144fa64f59bf3d0a30278bcaab2cR89